5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
@vpavlin so to summary:

* We don't know what is the story around Atomic labels (?) and it is fine for users to copy something they don't understand
* The current labels schema has some backward compatibility problems I highlighted here https://github.com/openshift/mysql/pull/83/files#r38512465 The problem with the 'docker command' is that is very hard to understand (at least for me, and I don't count as a regular user...)
* What you need to write to host here? You bind mount a directory where you write the database. This directory can be 0777 or in the worst case owned by arbitrary user (similar to OpenShift persistent volumes).
* If `OPTx` are mess and we all agree on that, then we should find a way to fix it.